pkg/util/net.go

```diff
@@ -26,3 +31,29 @@ func GetHostIP() (string, error) {
 	}
 	return ip.String(), nil
 }
+
+func RetryHttpGet(url string) int {
```

Comment:
Http->Https

Reply:
What about RetryInsecureHttpsGet ? So it's clear that we don't use any TLS certs.

Reply:
yes, let's go with that name